agent/xds/config.go

```diff
@@ -27,6 +27,12 @@ type ProxyConfig struct {
 	// Note: This escape hatch is compatible with the discovery chain.
 	PublicListenerJSON string `mapstructure:"envoy_public_listener_json"`
 
+	// LstenerTracingJSON is a complete override ("escape hatch") for the
+	// listeners tracing configuration.
```

Comment:
```suggestion
	// upstream listener's tracing configuration.
```

Reply:
This won't only affect the upstreams listeners but also the application listener. Should I make this change?

Reply:
Ah, I missed that. Disregard this then